lib/puppet/type/keycloak_protocol_mapper.rb

```diff
@@ -239,7 +240,14 @@ def self.title_patterns
   end
 
   validate do
-    if self[:protocol] == 'openid-connect' && !['oidc-usermodel-property-mapper', 'oidc-full-name-mapper', 'oidc-group-membership-mapper', 'oidc-audience-mapper'].include?(self[:type])
+    openid_connect_types = [
+      'oidc-usermodel-property-mapper',
+      'oidc-full-name-mapper',
+      'oidc-group-membership-mapper',
+      'oidc-audience-mapper',
+      'oidc-usermodel-attribute-mapper'
```

Comment:
Rubocop wants trailing comm

```suggestion
      'oidc-usermodel-attribute-mapper',
```